vault/expiration.go

```diff
@@ -245,7 +245,14 @@ func expireLeaseStrategyFairsharing(ctx context.Context, m *ExpirationManager, l
 		return
 	}
 
-	m.jobManager.AddJob(job, mountAccessor)
+	queueSize := m.jobManager.AddJob(job, mountAccessor)
+	metrics.SetGaugeWithLabels(
+		[]string{"expire", "lease_expiration", "pending"},
+		float32(queueSize),
+		[]metrics.Label{
+			{"mount_point", mountAccessor},
```

Comment:
Can we use mountpoint instead of mountAccessor here?  We have a bunch of other metrics that use mountpoint and none that use mountAccessor - plus users aren't likely to know what any mount accessors correspond to.

Reply:
Can you remind me what the differences are? If i'm remembering right, mount accessor is more in line with where the data physically lives, and mount point can be user defined (and i think can be not 1:1 related to a mount accessor).

The goal of this is to see which backends might be causing issues

Reply:
Mount accessor is a random UUID that doesn't change when you remount a backend on a new path (I don't recall if it's the same UUID that's used as a storage prefix).  Mountpoint is how the user thinks of the mount, i.e. the HTTP path prefix for requests that should go to that backend.

Reply:
can multiple mount points be associated with the same mount accessor?

Reply:
No